apps/web/cypress/tests/organization-brand.spec.ts

```diff
@@ -1,5 +1,6 @@
 describe('Organization Brand Screen', function () {
   beforeEach(function () {
+    cy.mockFeatureFlags({ IS_INFORMATION_ARCHITECTURE_ENABLED: true });
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** While this is good for one case, we also need the tests to confirm we haven't broken anything in the original flow‏

Reply:
The orginal flow has a different spec file `branding.spec.ts`

Reply:
Oh whoops sorry -- carry on!